From 5b6e415431e946e96265f29e3284a93c68da1562 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Wed, 4 Jan 2023 12:04:47 +0100
Subject: python3-setuptools-scm: upgrade 7.0.5 -> 7.1.0

Signed-off-by: Alexander Kanavin <alex@linutronix.de>
Signed-off-by: Alexandre Belloni <alexandre.belloni@bootlin.com>
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>

Backported to kirkstone

Signed-off-by: Jasper.Orschulko <Jasper.Orschulko@iris-sensing.com>
---
 .../python/python3-setuptools-scm_7.1.0.bb         | 30 ++++++++++++++++++++++
 1 file changed, 30 insertions(+)
 create mode 100644 meta/recipes-devtools/python/python3-setuptools-scm_7.1.0.bb

(limited to 'meta/recipes-devtools/python/python3-setuptools-scm_7.1.0.bb')

diff --git a/meta/recipes-devtools/python/python3-setuptools-scm_7.1.0.bb b/meta/recipes-devtools/python/python3-setuptools-scm_7.1.0.bb
new file mode 100644
index 0000000000..7961ff7198
--- /dev/null
+++ b/meta/recipes-devtools/python/python3-setuptools-scm_7.1.0.bb
@@ -0,0 +1,30 @@
+SUMMARY = "the blessed package to manage your versions by scm tags"
+HOMEPAGE = "https://pypi.org/project/setuptools-scm/"
+DESCRIPTION = "setuptools_scm handles managing your Python package versions in SCM metadata instead of declaring them as the version argument or in a SCM managed file."
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://PKG-INFO;beginline=8;endline=8;md5=8227180126797a0148f94f483f3e1489"
+
+SRC_URI[sha256sum] = "6c508345a771aad7d56ebff0e70628bf2b0ec7573762be9960214730de278f27"
+
+PYPI_PACKAGE = "setuptools_scm"
+inherit pypi python_setuptools_build_meta
+
+UPSTREAM_CHECK_REGEX = "setuptools_scm-(?P<pver>.*)\.tar"
+
+DEPENDS += "python3-tomli-native python3-packaging-native python3-typing-extensions-native"
+
+RDEPENDS:${PN} = "\
+    ${PYTHON_PN}-packaging \
+    ${PYTHON_PN}-pip \
+    ${PYTHON_PN}-pyparsing \
+    ${PYTHON_PN}-setuptools \
+    ${PYTHON_PN}-tomli \
+    ${PYTHON_PN}-typing-extensions \
+"
+
+RDEPENDS:${PN}:append:class-target = " \
+    ${PYTHON_PN}-debugger \
+    ${PYTHON_PN}-json \
+"
+
+BBCLASSEXTEND = "native nativesdk"
-- 
cgit v1.2


